src/components/calculate/CalculateForm.js

```diff
@@ -70,8 +88,8 @@ export default function CalculateForm(props) {
   useEffect(() => {
     formValues.map((variable) => {
       if (
-        variable.name === 'C1_PDRSAug24_PDRS__postcode' ||
-        variable.name === 'F7_PDRSAug24_PDRS__postcode'
+        variable.name === C1_PDRSAug24_PDRS__postcode ||
```

Comment:
notes: avoiding all the hardcoded values to prevent typos and understand the references